slang-antlr/src/main/java/com/sonarsource/slang/parser/SLangConverter.java

```diff
@@ -87,13 +87,6 @@
     SLangParser.THIS
   ));
 
-  private static final Set<Integer> LITERAL_TOKEN_TYPES = new HashSet<>(Arrays.asList(
-    SLangParser.IntegerLiteral,
-    SLangParser.BooleanLiteral,
-    SLangParser.CharacterLiteral,
```

Comment:
Other analyzers also consider character literal as strings for CPD. Up to you if you want to include it or not, since strictly the documentation only mention strings...

Reply:
Let's not consider character literals as strings. I don't think it would have real value.